src/components/FileTreeNode/index.spec.tsx

```diff
@@ -178,4 +196,28 @@ describe(__filename, () => {
 
     expect(renderChildNodes).toHaveBeenCalled();
   });
+
+  it('marks a file node as selected', () => {
+    const version = fakeVersion;
+    const store = configureStore();
+    store.dispatch(versionsActions.loadVersionInfo({ version }));
+    store.dispatch(
+      versionsActions.loadVersionFile({
+        path: version.file.selected_file,
+        version,
+      }),
+    );
+
+    const renderProps = getTreefoldRenderProps({
+      id: version.file.selected_file,
+    });
+
+    const root = render({
+      ...renderProps,
+      store,
+      versionId: version.id,
+    });
+
+    expect(root.find(`.${styles.selected}`)).toHaveLength(1);
```

Comment:
for paranoia, it would be nice to have a test that makes sure this style isn't applied when the file is not selected.